engine/src/flutter/impeller/display_list/aiks_dl_basic_unittests.cc

```diff
@@ -1591,5 +1591,30 @@ TEST_P(AiksTest, NoDimplesInRRectPath) {
   ASSERT_TRUE(OpenPlaygroundHere(callback));
 }
 
+TEST_P(AiksTest, BackdropFilterOverUnclosedClip) {
+  DisplayListBuilder builder;
+
+  builder.DrawPaint(DlPaint().setColor(DlColor::kWhite()));
```

Comment:
nit: I think this would be easier to follow if you added some curly braces around the save/restores.

Reply:
Done